Functions/Deploy-GomUser.ps1

```diff
@@ -27,21 +32,27 @@ function Deploy-GomUser {
 
     $IsMember = Test-GitHubOrganizationMember -OrganizationName $OrganizationName -UserName $UserName
     if($IsMember){
-        Write-Verbose "User '$UserName' is already a member of Organization '$OrganizationName'."
+        Write-Verbose "User '$UserName' is already a member of Organization '$OrganizationName'. Ensuring '$UserName' has $UserRole level permissions..."
+        $UpdateRole = @{
+            UriFragment = "orgs/$OrganizationName/memberships/$UserName"
+            Method = "Put"
+            Body = @{
+                role = $UserRole
+            } | ConvertTo-Json -Compress
+        }
+        Invoke-GHRestMethod @UpdateRole | Out-Null
         return
     } else {
-        # TODO: add support for add-as-admin/billing manager
-        # TODO: add teams
         $InviteUser = @{
             UriFragment = "orgs/$OrganizationName/invitations"
             Method  = 'Post'
             Body = @{
                 invitee_id = $User.id
-                role = "direct_member"
+                role = if($UserRole -eq "admin") {"admin"} else {"direct_member"}
```

Comment:
Is the `if/else` logic needed for a reason I'm missing? Or just an oversight?

```suggestion
                role = $UserRole
```

Reply:
Is this a `member` vs `direct_member` disambiguation thing?

Reply:
Yes, this is because the call to get the users role returns `member`, while the call to update the role accepts `direct_member`